File: futbolito-back/src/main/java/com/futbolito/security/dto/NewUser.java
package com.futbolito.security.dto;

import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class NewUser {
	
    @NotBlank
    private String name;
    
//    @NotBlank
//    private String lastName;
    
    @NotBlank
    private String nickname;
    
//    @NotBlank
//    private String phone;
    
    @Email
    private String email;
    
    @NotBlank
    private String password;
    
//    @NotBlank
//    private String confirmationPassword;

}

File: futbolito-front/src/app/models/new-user.ts
export class NewUser {

   name?: string;
    
   lastName?: string;
    
   nickname?: string;
    
   phone?: string;
    
   email?: string;
    
   password?: string;
    
   confirmationPassword?: string;

   constructor(name: string, lastName: string, nickname: string, phone: string,
       email: string, password: string, confirmationPassword: string) {
        this.name = name ;
        this.lastName = lastName ;
        this.nickname = nickname ;
        this.phone = phone ;
        this.email = email ;
        this.password = password ;
        this.confirmationPassword = confirmationPassword ;

   }
}
